moped-database/views/project_list_view.sql

```diff
@@ -15,7 +15,7 @@ CREATE OR REPLACE VIEW project_list_view AS WITH project_person_list_lookup AS (
 funding_sources_lookup AS (
     SELECT
         mpf_1.project_id,
-        string_agg(mfs.funding_source_name, ', '::text) AS funding_source_name
```

Comment:
@mddilley -- Sorry if it's strange to tag you in a PR that I'm actively working on, but I wanted to ask you --

I'm not 100% sure you asked for this, but we did circle around the idea, at least, during our conversation yesterday. Do you like this change? For the existing funding column where it creates a comma-delimited list of funding sources names, do you like this change where i now am deduplicating them and ordering alphabetically? 

FWIW, i know this issue is now written to add a new column and leave this one be, but since I'm in here .. 😉